lib/get-unified-config.js

```diff
@@ -65,7 +65,11 @@ function getUnifiedConfig () {
     reporter: {location: 'b', type: ['ssl', 'udp', 'file']},
     endpoint: {location: 'b', type: 's', name: 'COLLECTOR'},
     trustedPath: {location: 'b', type: 's', name: 'TRUSTEDPATH'},
-
+    unifiedLogging: {
+      location: 'b',
+      type: ['always', 'never', 'preferred'],
+      default: 'preferred'
```

Comment:
just curious ... what does 'preferred' mean?

Reply:
preferred means to try to use it (prefer to use it). it will be disabled if not compatible with the node version being executed.